source/agora/consensus/Fee.d

```diff
@@ -287,6 +287,7 @@ public class FeeManager
     private Amount[] getValidatorFees (Amount tot_fee, Amount tot_data_fee,
         UTXO[] stakes) nothrow @safe
     {
+        import std.numeric : gcd;
```

Comment:
Why the direct `gcd` usage instead of `Amount.gdc` ? Is it because `Amount.gcd` doesn't accept a range ?

Reply:
Because they are not `Amount`s anymore. They are plain `ulong`s, shares. `Amount.gcd()` was only used here, we might as well remove it.